src/helm/benchmark/scenarios/audio_language/audiocaps_senario.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+import os
+
+from helm.benchmark.scenarios.scenario import (
+    Scenario,
+    Instance,
+    Reference,
+    TEST_SPLIT,
+    CORRECT_TAG,
+    Input,
+    Output,
+)
+from tqdm import tqdm
+from datasets import load_dataset
+from helm.common.media_object import MediaObject, MultimediaObject
+from helm.common.general import ensure_file_downloaded
+
+
+class AudioCapsScenario(Scenario):
+    """AudioCaps
+
+    AudioCaps is a large-scale dataset of about 46K audio clips to human-written text pairs collected
+    via crowdsourcing on the AudioSet dataset, which covers a wide range of human and animal sounds,
+    musical instruments and genres, and common everyday environmental sounds.
+
+    Paper: https://aclanthology.org/N19-1011.pdf
+    Code: https://github.com/cdjkim/audiocaps
+
+    Citation:
+    @inproceedings{audiocaps,
+        title={AudioCaps: Generating Captions for Audios in The Wild},
+        author={Kim, Chris Dongjoo and Kim, Byeongchang and Lee, Hyunmin and Kim, Gunhee},
+        booktitle={NAACL-HLT},
+        year={2019}
+        }
+    """
+
+    name = "audiocaps"
+    description = "A large-scale dataset of about 46K audio clips to human-written text pairs \
+        ([Kim et al, 2019](https://aclanthology.org/N19-1011.pdf))."
+    tags = ["audio", "captioning"]
+
+    def get_instances(self, output_path: str) -> List[Instance]:
+        instances: List[Instance] = []
+        data_dir: str = os.path.join(output_path, "wav_files")
+
+        ensure_file_downloaded(
+            source_url="https://huggingface.co/datasets/Olivia714/audiocaps/resolve/main/wav_files.zip",
+            target_path=data_dir,
+            unpack=True,
+        )
+
+        assert os.path.exists(data_dir), "Download the wav_files from Olivia714/audiocaps"
+
+        for row in tqdm(load_dataset("Olivia714/audiocaps", cache_dir=output_path, split=TEST_SPLIT)):
```

Comment:
Can we use`load_dataset(self.HF_DATASET_NAME)` and `HF_DATASET_NAME="Olivia714/audiocaps"` instead? to make it compatible with other scenarios.